doxygen/src/apis_schema.txt

```diff
@@ -171,11 +172,17 @@ by either implementing or using a set of functions, data types, and definitions.
   </tr>
   <tr>
     <td>Capiversion</td>
-    <td>Version of this component. Is a mandatory part of the component ID.  The version format is described in \ref VersionType "Version Type".
+    <td>Version of the api. Is a mandatory part of the api ID.  The version format is described in \ref VersionType "Version Type".
 	</td>
     <td>VersionType</td>
     <td>optional</td>
   </tr>
+  <tr>
+    <td>licenseSet</td>
+	<td>Reference to the licenseSet with the given identifier \token{ID} listed in the licenseSets section of the package description.
+	    A licenseSet describes one or more license files which in this case govern the api.</td>
```

Comment:
This may need your attention, @jkrech.

Reply:
@LMESTM just to be clear, this is the <api> definition section and since such an API maybe governed by a license which is different from the implementations of such an API, I think we should allow to specify a license reference for the api.

e.g. CMSIS-Driver USART API is specified and the header file is stored in  ARM.CMSIS pack using Apache 2.0 license
The implementation of a CMSIS-Driver USART for a device maybe governed by a different license in respective DFP.

Reply:
@jkrech was my mistake when reading, I though this was the licence for any component ... as this is the api section yes the description is valid. Please discard my comment